.gitbook/contents/hc/1_write-handler.md

```diff
@@ -0,0 +1,152 @@
+# Write an Off-Chain Handler
+
+The first step is to write our *off-chain handler*. This handler is responsible for receiving requests from the bundler along with their payloads and returning the appropriate responses.
+
+## Simple Example
+
+Let's begin with a simple example where the handler receives two numbers. It will perform both addition and subtraction on these numbers. If the result of the subtraction results in an overflow (i.e. the first number is greater than the second), the handler will respond with an underflow error.
```

Comment:
This line should say that an error will be generated if there is an underflow in the subtraction, when "b" is larger than "a" in the (a-b) expression.